pkg/server/server_test.go

```diff
@@ -24,7 +24,7 @@ func TestServerRootPost(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 	w := httptest.NewRecorder()
-	req, _ := http.NewRequest(http.MethodPost, "/", bytes.NewBufferString("hello"))
+	req, _ := http.NewRequest(http.MethodPost, "/proxy", bytes.NewBufferString("hello"))
```

Comment:
We really need to test both paths - `/` and `/proxy`. Since we know `/` will be going away, I think it is fine to copy the code into a new method.